src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/NomisServiceImpl.java

```diff
@@ -213,11 +212,11 @@ public List<CaseNoteUsageDto> getCaseNoteUsage(final List<Long> staffIds, final
                 .numMonths(numMonths)
                 .build();
 
-        return restCallHelper.post(CASE_NOTE_USAGE, queryParamsOf(), uriVariablesOf(), body, CASE_NOTE_USAGE_DTO_LIST, false);
+        return restCallHelper.post(CASE_NOTE_USAGE, queryParamsOf(), uriVariablesOf(), body, CASE_NOTE_USAGE_DTO_LIST, true);
```

Comment:
true means use system-client, false means pass through user token

Reply:
If only it was Kotlin with a named parameter!